src/store.cc

```diff
@@ -772,6 +777,16 @@ StoreEntry::write (StoreIOBuffer writeBuffer)
     invokeHandlers();
 }
 
+void
+StoreEntry::writeData(StoreIOBuffer writeBuffer)
+{
+    assert(mem_obj);
+    debugs(20, 6, "offset " << writeBuffer.offset << " len " << writeBuffer.length);
```

Comment:
If this method survives, to simplify, reduce maintenance costs, and provide more info:

```suggestion
    debugs(20, 6, writeBuffer);
```